sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -197,6 +199,84 @@ public async Task UpdateCapabilitiesAsync()
             Assert.AreEqual(200, updateOperation.GetRawResponse().Status);
         }
 
+        [Test]
+        public async Task ReleaseUnathorizedNumber()
+        {
+             const string PhoneNumber = "+14255550123";
+            var client = CreateClient();
+            try
+            {
+                var releaseOperation = await client.StartReleasePhoneNumberAsync(PhoneNumber);
+            }catch (Exception e)
+            {
+                Assert.NotNull(e);
+            }
+        }
+
+        [Test]
+        public async Task UpdateCapabilitiesUnathorizedNumber()
+        {
+            const string PhoneNumber = "+14255550123";
+            var capabilities = new PhoneNumberCapabilities(calling: PhoneNumberCapabilityType.None, sms: PhoneNumberCapabilityType.Outbound);
+            var client = CreateClient();
+            try
+            {
+                var UpdateCapabilitiesOperation = await client.StartUpdateCapabilitiesAsync(PhoneNumber);
+            }
+            catch (Exception e)
+            {
+                Assert.NotNull(e);
+            }
+        }
+
+        [Test]
+        public async Task GetPurchasedUnathorizedNumber()
+        {
+            var client = CreateClient();
+            const string PhoneNumber = "+14255550123";
+            try
+            {
+                var purchaseOperation = await client.GetPurchasedPhoneNumberAsync(PhoneNumber);
+            }
+            catch (Exception e)
+            {
+                Assert.NotNull(e);
+            }
+        }
+
+        [Test]
+        public async Task StartPurchasedUnathorizedNumber()
+        {
+            var client = CreateClient();
+            const string PhoneNumber = "+14255550123";
+
+            try
+            {
+                var releaseOperation = await client.StartPurchasePhoneNumbersAsync(PhoneNumber);
+            }
+            catch (Exception e)
+            {
+                Assert.NotNull(e);
+            }
+        }
+
+        [Test]
+        public async Task GetPurchasedPhoneNumbersNextPage()
```

Comment:
I thought we already have a test case called `GetPurchasedPhoneNumbers` which does the same thing. What additional value is intended for this new case?

Reply:
We never tested for the Next page as normally our test cases does not have any nextLink in the response here in this test case the json has a nextLink so is actually covering the next page